Create agentic training document

GuidoBR opened the pull request.

GuidoBR committed:
Create agentic training document

GuidoBR committed:
Tools notice

wesleyfuchter commented:
@GuidoBR we also need to copy this file to confluence, similar to this https://moduscreate.atlassian.net/wiki/spaces/QAD/pages/6372163626/GitHub+Copilot+Intermediate+-+201

arobertn commented on TRAINING.md:
```diff
@@ -0,0 +1,194 @@
+# GitHub Copilot Agent Mode - Advanced Training
```
File should be called "TRAINING.md" (remove one 'N').

wesleyfuchter replied:
@copilot can you handle this?

Copilot committed:
Rename TRAINNING.md to TRAINING.md

Agent-Logs-Url: https://github.com/tweag/agentic-coding-handbook/sessions/a67d0b60-d0af-4e32-9220-ce02f1d7b89b

Co-authored-by: wesleyfuchter <12664143+wesleyfuchter@users.noreply.github.com>
